source/_docs/manual-import.md

```diff
@@ -56,7 +56,7 @@ As long as you've chosen the same codebase (Drupal 7, Commerce Kickstart, etc.)
  git remote add pantheon <ssh_url>
  ```
 
-5. **Drupal only**: Move your settings file to `settings.local.php` so that it will be ignored by Git and included from Pantheon's `settings.php`. First, make sure that you can modify it, and restore the protections after the move:
+5. **Drupal only**: Move your settings file to `settings.local.php` and add it to `.gitignore` so that it will be ignored by Git and included from Pantheon's `settings.php`. Make sure that you can modify it, and restore the protections after the move:

```

Comment:
Suggest removing this step - or clarifying the context so that the user knows how/when to reintroduce their configurations on Pantheon.